react/features/base/label/components/web/Label.tsx

```diff
@@ -92,13 +92,27 @@ const Label = ({
 }: IProps) => {
     const { classes, cx } = useStyles();
 
+    const onKeyPress = useCallback(event => {
+        if (!onClick) {
+            return;
+        }
+
+        if (event.key === 'Enter' || event.key === ' ') {
+            event.preventDefault();
+            onClick();
+        }
+    }, [ onClick ]);
+
     return (
         <div
             className = { cx(classes.label, onClick && classes.clickable,
                 color && classes[color], className
             ) }
             id = { id }
-            onClick = { onClick }>
+            onClick = { onClick }
+            onKeyPress = { onKeyPress }
+            role = { onClick ? 'button' : undefined }
```

Comment:
I'd also argue here that `undefined` would be better. The idea here is to say, "view this as a button if it's interactive, do as usual if not". `role="none"` removes the usual `role`. And I'd say it's better to have the usual role instead of `status` that can trigger automatic stuff depending on screen readers, something I think we wouldn't want in this low level component that could be used for different things.